rust/chains/hyperlane-sealevel/src/multisig_ism.rs

```diff
@@ -136,6 +137,7 @@ impl SealevelMultisigIsm {
 
 mod contract {
```

Comment:
sorry for not flagging this earlier -- now that the workspaces are joined, we can also remove this entire `mod contract`. similar to `./mailbox.rs` in here, `mod contract` was a copy-paste workaround to get some types from the programs into here, and we can now import from the programs directly

e.g. `ValidatorsAndThreshold` below can be found here https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/b86e00076c909d327cb41887f5add7ea863ed1e8/rust/sealevel/programs/ism/multisig-ism-message-id/src/instruction.rs#L65. Should be able to just search for the types and import them as needed but lmk if there are issues